VideoLocker/src/main/java/org/edx/mobile/util/SecurityUtil.java

```diff
@@ -0,0 +1,50 @@
+package org.edx.mobile.util;
+
+
+import android.content.Context;
+import android.content.pm.PackageInfo;
+import android.content.pm.PackageManager;
+import android.support.annotation.NonNull;
+
+import org.edx.mobile.logger.Logger;
+import org.edx.mobile.module.db.DbStructure;
+import org.edx.mobile.module.prefs.PrefManager;
+
+import java.io.File;
+import java.util.Collections;
+
+public abstract class SecurityUtil {
+    private static final Logger logger = new Logger(SecurityUtil.class);
+
+    public static void clearUserData(@NonNull Context context) {
+        // Clear the App's directory
+        PackageManager packageManager = context.getPackageManager();
+        try {
+            PackageInfo packageInfo = packageManager.getPackageInfo(context.getPackageName(), 0);
+            File appDir = new File(packageInfo.applicationInfo.dataDir);
+            if (appDir.exists() && appDir.isDirectory()) {
+                File[] children = appDir.listFiles();
+                for (int size = children.length, i = 0; i < size; i++) {
```

Comment:
This can be simplified by converting it to a foreach loop. Note that using foreach loop in arrays don't even have the (negligible) performance penalty of having to create an `Iterator`.

```` java
for (final File child : appDir.listFiles())
````